### PR TITLE
ActionMenu stories

### PR DESCRIPTION
**Pull request by langermank**
As part of https://github.com/github/primer/issues/1488

### Merge checklist

- [ ] Added/updated tests
- [ ] Added/updated documentation
- [ ] Tested in Chrome
- [ ] Tested in Firefox
- [ ] Tested in Safari
- [ ] Tested in Edge

### CODE REVIEW

File: src/__tests__/ActionMenu.test.tsx [hunk 1]
```diff
@@ -113,10 +114,10 @@ describe('ActionMenu', () => {
   it('should be able to select an Item with selectionVariant', async () => {
     const component = HTMLRender(
       <ThemeProvider theme={theme}>
-        <SingleSelection />
+        <SingleSelect />
       </ThemeProvider>,
     )
-    const button = component.getByLabelText('Field type')
+    const button = component.getByRole('button')
```
Comment (langermank) [inline src/__tests__/ActionMenu.test.tsx:120]: Can someone help me improve this test? Im not sure how to select this button as I've made the button label dynamic.

Comment (colebemis) [reply src/__tests__/ActionMenu.test.tsx:120]: You could use a regex
```suggestion
    const button = component.getByLabelText(/^options/i)
```

**Review by colebemis — APPROVED** [review]
🎉

File: generated/components.json [hunk 2]
```diff
@@ -1943,6 +1857,97 @@
...
+      "a11yReviewed": true,
+      "stories": [
+        {
+          "id": "components-actionmenu--default",
+          "code": "() => (\n  <ActionMenu>\n    <ActionMenu.Button>Open menu</ActionMenu.Button>\n    <ActionMenu.Overlay width=\"medium\">\n      <ActionList>\n        <ActionList.Item onSelect={() => alert('Copy link clicked')}>\n          Copy link\n          <ActionList.TrailingVisual>⌘C</ActionList.TrailingVisual>\n        </ActionList.Item>\n        <ActionList.Item onSelect={() => alert('Quote reply clicked')}>\n          Quote reply\n          <ActionList.TrailingVisual>⌘Q</ActionList.TrailingVisual>\n        </ActionList.Item>\n        <ActionList.Item onSelect={() => alert('Edit comment clicked')}>\n          Edit comment\n          <ActionList.TrailingVisual>⌘E</ActionList.TrailingVisual>\n        </ActionList.Item>\n        <ActionList.Divider />\n        <ActionList.Item\n          variant=\"danger\"\n          onSelect={() => alert('Delete file clicked')}\n        >\n          Delete file\n          <ActionList.TrailingVisual>⌘D</ActionList.TrailingVisual>\n        </ActionList.Item>\n      </ActionList>\n    </ActionMenu.Overlay>\n  </ActionMenu>\n)"
```
Comment (colebemis) [inline generated/components.json:1868]: 🎉

**Pull request merged**